test/e2e/e2e.go

```diff
@@ -50,3 +54,65 @@ func CreateRouteAndConfig(clients *test.Clients, logger *logging.BaseLogger, ima
 	err := test.CreateRoute(logger, clients, names)
 	return names, err
 }
+
+// TearDownByYaml will delete resouce defined in yaml.
+func TearDownByYaml(yamlFilename string, logger *logging.BaseLogger) {
+	if yamlFilename != "" {
+		exec.Command("kubectl", "delete", "-f", yamlFilename).Run()
+		os.Remove(yamlFilename)
+	}
+}
+
+// CreateConfigAndRouteFromYaml will create Route and Config by yaml file.
+func CreateConfigAndRouteFromYaml(
```

Comment:
Can we create/delete using the same methods as other tests?

If we are going to change how we deploy test stuff, I'd rather we do it holistically (and not in this already large change).  cc @dgerd

Reply:
I think it had cover the all tests, Only `TestHelloWorldFromShell` and `TestHelloWorldUserPort` create resource by yaml now.